ui/webui/resources/brave_webui_resources.grd

```diff
@@ -12,10 +12,10 @@
   <release seq="1">
     <includes>
       <part file="fonts_resources.grdp" />
-      <if expr="not is_android and not is_ios">
+      <if expr="not is_ios">
         <part file="page_specific_v3.grdp" />
       </if>
-      <if expr="not is_android and not is_ios">
+      <if expr="not is_ios">
         <part file="br_elements_resources_v3.grdp" />
```

Comment:
do we really need two grdp files above for Android?

Reply:
Fixed (excluded both from the android build)